client/my-sites/scan/controller.js

```diff
@@ -100,9 +100,9 @@ export function scan( context, next ) {
 export function scanHistory( context, next ) {
 	const state = context.store.getState();
 	const siteId = getSelectedSiteId( state );
-	const showNavigation = isJetpackCloud() || ! isAtomicSite( state, siteId );
+	const showScanNavigation = ! isAtomicSite( state, siteId ) && isJetpackCloud();
```

Comment:
Nice 👍 doesn't go to the unavailable Scanner link.
I am curious if this is due to we auto-run Jetpack Scan for Atomic sites, compared to Jetpack sites.

Reply:
yes that's it! Scanning is done automatically, we just need to show them history of scans done to make the feature more visible to the user

Reply:
Got it, thanks for the clarification!